src/app/public/modules/flyout/flyout.component.ts

```diff
@@ -251,6 +251,18 @@ export class SkyFlyoutComponent implements OnDestroy, OnInit {
     this.adapter.toggleIframePointerEvents(true);
   }
 
+  public onIteratorPreviousClick() {
+    if (this.config.iterator && !this.config.iterator.previousButtonDisabled) {
```

Comment:
Yeah true, but these are public methods. I think I was just adding extra safeguards here in case they were used for something else in the future. Could I persuade you on this one in the interest of being extra cautious?